tripleblindmarket/private-kit
# Qa feedbacks user policy

#### Description:

Term of Service font size adjustment in Spanish.

#### Linked issues:

[Related ticket](https://trello.com/c/OtqapT8i/147-incluir-las-pol%C3%ADticas-de-uso-de-usuarios)

#### Screenshots:



#### How to test:

Reinstall the app to get the onboarding message and see the change. Must be in spanish
